fastlane/download_metadata.swift

```diff
@@ -37,9 +56,13 @@ let languages = [
     "zh-Hant": "zh-tw",
 ]
 
-func downloadTranslation(languageCode: String, folderName: String) {
+func downloadTranslation(
+    config: Config = .config(for: CommandLine.arguments.second),
```

Comment:
There's actually a more idiomatic way of doing this arguments management in the form of [`ArgumentParser`](https://swift.org/blog/argument-parser/) but it seemed out of scope for the context of this PR.

Reply:
That would require us to use a `Package.swift` to pull the ArgumentParser package first via SwiftPM to be then able to use it, tho 😓